FindServers fails because client is not connected

When `UaTcpStackClient.findServers(endpoint)` is called, it always fails with an exception:

```
java.util.concurrent.ExecutionException: UaException: status=Bad_ServerNotConnected, message=The operation could not complete because the client is not connected to the server.
	at java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:357)
	at java.util.concurrent.CompletableFuture.get(CompletableFuture.java:1895)
[...]
Caused by: UaException: status=Bad_ServerNotConnected, message=The operation could not complete because the client is not connected to the server.
	at org.eclipse.milo.opcua.stack.core.util.FutureUtils.failedUaFuture(FutureUtils.java:100)
	at org.eclipse.milo.opcua.stack.core.util.FutureUtils.failedUaFuture(FutureUtils.java:88)
	at org.eclipse.milo.opcua.stack.client.ClientChannelManager.getChannel(ClientChannelManager.java:226)
	at org.eclipse.milo.opcua.stack.client.UaTcpStackClient.sendRequest(UaTcpStackClient.java:182)
	at org.eclipse.milo.opcua.stack.client.UaTcpStackClient.findServers(UaTcpStackClient.java:555)
```

--- a/opc-ua-stack/stack-client/src/main/java/org/eclipse/milo/opcua/stack/client/UaTcpStackClient.java
+++ b/opc-ua-stack/stack-client/src/main/java/org/eclipse/milo/opcua/stack/client/UaTcpStackClient.java
@@ -551,10 +551,12 @@ public class UaTcpStackClient implements UaStackClient {
         FindServersRequest request = new FindServersRequest(
             new RequestHeader(null, DateTime.now(), uint(1), uint(0), null, uint(5000), null),
             endpointUrl, null, null);
-
-        return client.<FindServersResponse>sendRequest(request)
-            .whenComplete((r, ex) -> client.disconnect())
-            .thenApply(FindServersResponse::getServers);
+        
+        return client.connect().thenCompose(c ->
+            c.<FindServersResponse>sendRequest(request)
+                .whenComplete((r, ex) -> client.disconnect())
+                .thenApply(FindServersResponse::getServers)
+        );
     }
 
     /**

